polkadot/node/core/backing/src/tests/mod.rs

```diff
@@ -299,6 +295,81 @@ async fn test_startup(virtual_overseer: &mut VirtualOverseer, test_state: &TestS
 	);
 }
 
+async fn test_validation_requests(
```

Comment:
I'd name it `assert_*` instead of `test_*` since it's not a test on its own, but a part of expectations/assertion.

Reply:
Fixed